multimedia/M2VDownsizer/Files/Makefile

```diff
@@ -0,0 +1,257 @@
+# Makefile for M2VDowsizer
+#
+# Author:   Robert Kennedy
+# EMAIL:    amtor@hotmail.com
+#
+# License for Makefile:  GPL-2
+#
+# 'make depend' uses makedepend to automatically generate dependencies 
+#               (dependencies are added to end of Makefile)
+# 'make'        build executable file 'M2VDownsizer'
+# 'make clean'  removes all .o and executable files
+#
+#  Note:    There is NO "make install"
+#           After running "make", just copy M2VDownsizer into the desired directory
+#           e.g.    sudo chmod +x ./M2VDownsizer
+#                   sudo cp M2VDownsizer /usr/local/bin
+#               
+
+# Define the C compiler to use
+#
+CC := clang
```

Comment:
Is there a reason why you want to specify clang here? The default CC for make is `cc` which is sufficient for most projects, and on Mac OS X 10.7 and later, `cc` is clang, and MacPorts will override CC anyway.

Reply:
No reason other than I was having more trouble with gcc early on with this Project.  It could safely be changed to CC := cc if you so desire.

Reply:
Verified.  I changed `CC := clang` to `CC := clang` in the Makefile and M2VDownsizer built successfully.  

The binary builds fine using Macports or using the Makefile outside the Macports build environment.

Reply:
> I changed `CC := clang` to `CC := cc` in the Makefile

You can do that if you wish but it is pointless since `make` sets CC to `cc` by default.

Reply:
I changed `CC := clang` to `CC := cc` in my last push even though as pointed out it really does not make any difference since MacPorts will over-ride `CC:`  anyway.